docker-compose.yml

```diff
@@ -101,7 +110,8 @@ services:
         build: ./isb/
         depends_on:
             - isamples_inabox
-            - solr
+            solr:
+                condition: service_healthy
```

Comment:
The solr schema creation script also needs to wait until solr is healthy.  I saw this cause problems in the past, so it's nice we have a fix here too.

Reply:
I thought about this one a bit more, and this makes no sense.  The URL we use for the health check includes the collection name, which won't exist at this point so this could never succeed.  I am going to revert this change.